aait/backend/group-2A/Domain/Entities/Comment.cs

```diff
@@ -8,8 +8,8 @@ public class Comment
         //The User That Commented
         public required int UserId { get; set; }
         public required int PostId { get; set; }
-        public required int CreatedAt{ get; set; }
-        public required int UpdatedAt{ get; set; }
+        public required DateTime CreatedAt{ get; set; }
+        public required DateTime UpdatedAt{ get; set; }
         
         //Navigation Property
         public virtual User? User{ get; set; }
```

Comment:
If there is no derived class from Comment, there won't be a class to override the virtual property . so you can remove the virtual keyword